MovieApp/DataLayer/DataSources/MovieUserDefaultsDataSource/MovieUserDefaultsDataSourceProtocol.swift

```diff
@@ -0,0 +1,7 @@
+protocol MovieUserDefaultsDataSourceProtocol {
+
+    func toggleFavorited(for movieId: Int)
+
+    func getFavorites() -> [Int]
```

Comment:
functions that have no arguments and return a value should be computed properties